tests/unittest_brain_numpy_core_numerictypes.py

```diff
@@ -341,6 +345,59 @@ def test_datetime_astype_return(self):
             ),
         )
 
+    @unittest.skipUnless(
```

Comment:
I think we can add a small test to check that we're not crashing if numpy is'nt installed.

Reply:
@Pierre-Sassoulas i'm sorry but i don't see the point here. Can you explain a bit what you want to prevent?

Reply:
I think it would have detected the problem of the other remark (TypeError: '>' not supported between instances of 'NoneType' and 'tuple'), by having automated tests for it.

Reply:
@Pierre-Sassoulas i was thinking to add something alone the following lines:
```
@unittest.skipIf(HAS_NUMPY)
class NumpyBrainUtilsTest(unittest.TestCase):
    def test_get_numpy_version_do_not_crash(self):
        """
        Test that the function _get_numpy_version doesn't crash even if numpy is not installed
        """
        self.assertEqual(_get_numpy_version(), ('0', '0', '0'))
```
Is it what you were thinking of? Or do you had in mind something more generic?

Reply:
Looks good, but we could even check what we get when parsing the code without numpy (that would execute the ``_get_numpy_version`` too, but also much more code).